Makefile

```diff
@@ -127,4 +127,4 @@ pdf-user:
 .PHONY: check-prose
 check-prose:
 	@echo "Checking quality of the prose in all files"
-	write-good --parse **/*.adoc
+	@yarn prose
```

Comment:
How does yarn prose know which files to scan ?

Reply:
ah, just seen that it is defined in package.json/scripts

Reply:
What's `yarn prose`?

Reply:
Just looked at other files and see what this is about.